satpy/tests/reader_tests/test_fci_l1c_fdhsi.py

```diff
@@ -400,6 +422,52 @@ def test_load_bt(self, reader_configs, caplog):
             else:
                 numpy.testing.assert_array_almost_equal(res[ch], 209.68274099)
 
+    def test_load_index_map(self, reader_configs):
+        """Test loading of index_map."""
+        from satpy.readers import load_reader
+
+        filenames = [
+            "W_XX-EUMETSAT-Darmstadt,IMG+SAT,MTI1+FCI-1C-RRAD-FDHSI-FD--"
+            "CHK-BODY--L2P-NC4E_C_EUMT_20170410114434_GTT_DEV_"
+            "20170410113925_20170410113934_N__C_0070_0067.nc"
+        ]
+
+        reader = load_reader(reader_configs)
+        loadables = reader.select_files_from_pathnames(filenames)
+        reader.create_filehandlers(loadables)
+        res = reader.load(
+            [name + '_index_map' for name in
+             self._chans["solar"] + self._chans["terran"]], pad_data=False)
+        assert 16 == len(res)
+        for ch in self._chans["solar"] + self._chans["terran"]:
+            assert res[ch + '_index_map'].shape == (200, 11136)
+            numpy.testing.assert_array_equal(res[ch + '_index_map'][1, 1], 5138)
+
+    def test_load_aux_data(self, reader_configs):
+        """Test loading of index_map."""
+        from satpy.readers import load_reader
+        from satpy.readers.fci_l1c_fdhsi import AUX_DATA
+
+        filenames = [
+            "W_XX-EUMETSAT-Darmstadt,IMG+SAT,MTI1+FCI-1C-RRAD-FDHSI-FD--"
+            "CHK-BODY--L2P-NC4E_C_EUMT_20170410114434_GTT_DEV_"
+            "20170410113925_20170410113934_N__C_0070_0067.nc"
+        ]
+
+        reader = load_reader(reader_configs)
+        loadables = reader.select_files_from_pathnames(filenames)
+        reader.create_filehandlers(loadables)
```

Comment:
these lines could make a small function and be factorized across the tests.

Reply:
true, thanks!